### PR TITLE
Check if entity looked at has valid hyperlink href attached

### PR DESCRIPTION
**Pull request by venkatn93**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9a193acfa092030937c737f778a2079eca2adb75 available at: https://gist.github.com/7c13db01e0e51b1a6bc9<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5108.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5108.dmg

File: libraries/entities-renderer/src/EntityTreeRenderer.cpp [hunk 1]
```diff
@@ -919,7 +919,14 @@ void EntityTreeRenderer::mouseMoveEvent(QMouseEvent* event, unsigned int deviceI
...
         }
 
-        //qCDebug(entitiesrenderer) << "mouseMoveEvent over entity:" << rayPickResult.entityID;
+        QString urlString = rayPickResult.properties.getHref();
+        QUrl url = QUrl(urlString, QUrl::StrictMode);
+        if (url.isValid() && !url.isEmpty()){
+            qCDebug(entitiesrenderer) << "mouseMoveEvent over entity:" << urlString;
+        } else {
+            qCDebug(entitiesrenderer) << "mouseMoveEvent over entity:" << "Not valid href";
+        }
```
Comment (Atlante45) [inline libraries/entities-renderer/src/EntityTreeRenderer.cpp]: Can you guard that if with an `#ifdef WANT_DEBUG`?
Printing out debug is really slow and this gets called a lot.

**Comment by jherico** [comment]
I dont' really think we want this.  Rather than triggering on mouse move, we should be working with entityHoverEnter and entityHoverLeave.  See https://github.com/highfidelity/hifi/pull/5112

**Comment by hifi-gustavo** [comment]
Build console output for commit: b9b8cfd60d8f12f2c9b9c02e28bb959e91b4de47 available at: https://gist.github.com/7c13db01e0e51b1a6bc9<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5108.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5108.dmg

**Comment by venkatn93** [comment]
I guess I could remove my code and just make another PR with my latest progress

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7142be07b69a802d225adf735e0165b0ea887212 available at: https://gist.github.com/7c13db01e0e51b1a6bc9<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5108.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5108.dmg

**Pull request closed without merging**